css/hw3.css

```diff
@@ -0,0 +1,77 @@
+* {
+    box-sizing: border-box;
+    font-size: 14px;
+}    
+.header-main{
+    text-align: center;
+    margin-bottom: 100px;
+}
+.header-list{
+    display: inline-block;
+    margin: 0px;
+    padding: 0px;
+}
+.header-item{
+    display: inline-block;
+    margin-right: 123px;
+    margin-top: 77px;
+}
+.header-item a{
+    text-decoration: none;
+    color: #626262;
+    text-transform: uppercase;
+    font-size: 14px;
+}
+.last-header-item{
+    margin-right: 0px;
+}
+.header-item a:hover{
+    color: #03a9f4;
+}
+main{
+    text-align: center;
+}
+.main-list{ 
+    padding: 0px;
+    margin: 0px;
+    width: 970px;
+    display: inline-block;
+}
+.main-item{
+    display: inline-block;
+    width: 420px;
+    height: 480px;
+    overflow: hidden;
+    margin: 0px 30px 140px;
+}
+figcaption{
+    text-align: left;
+    margin: 30px 0px 25px;
+    text-transform: uppercase;
+    color: #b4ad9e;
+}
+.main-heading{
+    text-align: left;
+    font-size: 30px;
+    color: #626262;
+    margin-bottom: 25px;    
+}
+.main-text{
+    text-align: left;
+    color: #626262;
+    line-height: 24px;
+}
+.last-main-item{
```

Comment:
bad solution/ use https://developer.mozilla.org/ru/docs/Web/CSS/:nth-last-child fo this example